about.html

```diff
@@ -0,0 +1,198 @@
+                foundation activists, art galleries, libraries, museums,
+                archives, governments, foundations, lawyers, social activists, etc.The Forum strives in all its efforts
+                to demonstrate
+                entrepreneurship in the global public interest while upholding the
+                highest standards of governance.
+            </p>
+            <p class="questions">Please contact us via this email have any other questions
+                about SPHERE Global Forum!<br>
+                <br><a id="join" href="#">sphereglobal2022@sphere.org</a>
+            </p>
+        </div>
+
+    </header>
+    <main class="main">
+        <div class="main-about">
+
+            <h2 class="main-heading">Sphere Global Entrepreneurship Forum 2022</h2>
+            <hr class="hr-line1">
+
+            <p class="about-descr1">This is the new logo of Sphere Global which was decided last year.</p>
+
+            <div class="new-logo-container">
+                <img class="new-logo" src="./assets/logos/headerlg.png" alt="seminar logo">
+                <div class="logo-newname">
+                    <p>SPHERE </p>
+                    <p>Global Seminar</p>
+                </div>
+            </div>
+        </div>
+
+        <hr class="hr-mainabout">
+
+        <div class="about-infor" id="news">
+            <h2>View past Global Forums</h2>
+            <hr class="hr-line1">
+            <p class="view-descrip">Take a look at the recent global summits held in Bungoma, Mombasa and Malindi,
+                Diani.</p>
+            <div class="past-forum">
+
+                <div class="past-img1">
+                    <h3 class="past-heading1">2020</h3>
+                    <p class="past-desp1"> SPHERE GLOBAL Forum 2020 in Berlin</p>
+
+                </div>
+
+                <div class="past-img2">
+                    <h3 class="past-heading2">2018</h3>
+                    <p class="past-desp2"> SPHERE GLOBAL Forum 2018 in Cairo</p>
+                </div>
+            </div>
+        </div>
+    </main>
+
+    <footer class="footer">
+        <div class="top-footer">
+            <div class="sponsors">
+                <h2>Sponsors</h2>
+                <hr class="hr-line1">
+            </div>
+            <ul class="sponsor-list1">
+                <li><a class="sponsor-link one" href="#">mozilla</a></li>
+
+                <li><a class="sponsor-link two" href="#">Google</a></li>
+
+                <li><a class="sponsor-link three" href="#">NAVER</a></li>
+            </ul>
+
+            <ul class="sponsor-list2">
+
+                <li><a class="sponsor-link four" href="#">daumkakao</a></li>
+
+                <li><a class="sponsor-link five" href="#">
+                        <span class="iconify" data-icon="arcticons:airbnb" data-width="40" data-height="45"></span>
+                        airbnb</a>
+                </li>
+
+
+            </ul>
```

Comment:
- [ ] Your sponsor images not showing in both about and index page. Please kindly fix this issue as you showed in your video presentation

Screenshot
![image](https://user-images.githubusercontent.com/69011963/176915546-0abaa8f3-9d32-4ebb-ab4a-fa5468ba60f1.png)